.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# For more info about the settings, please refre to the github repository:
+# https://github.com/canonical/gh-jira-sync-bot
+#
+
+settings:
+  # Jira project key to create the issue in
+  jira_project_key: "SOLENG"
+
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: done
+
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  components:
+    - hardware-observer
```

Comment:
I think this should be changed to dcgm-snap. Can you confirm @aieri

Reply:
There is no such component; I can create one, but thought we could use `hardware-observer` as it already exists

Reply:
Components are free, so I have created the `dcgm` one.